docs/getting_started_guide/Installing_SIMP_From_A_Repository.rst

```diff
@@ -218,6 +218,11 @@ your clients.
   This would be the general technique that you would use to auto-bootstrap your
   clients via ``user-data`` scripts in cloud environments.
 
+.. NOTE::
+  If you didn't enable the ``simp::server::kickstart`` class on your server to
```

Comment:
It seems a little more logical to me that the text would explain up front that there are two SIMP-supported, client bootstrap options:
  - bootstrap from simp server acting as a kickstart server -> refer to existing documentation elsewhere on how to set this up
  - bootstrap from a non-SIMP kickstart server, using the runpuppet script -> give explicit description on how to generate that file